### PR TITLE
Follow up Lib9c & Libplanet's new rendering API

### PR DESCRIPTION
**Pull request by dahlia**
See also the patches merged into lib9c as well: <https://github.com/planetarium/lib9c/pull/23> & <https://github.com/planetarium/lib9c/pull/26>.

### CODE REVIEW

**Review by longfin — APPROVED** [review]

File: NineChronicles.Standalone/StandaloneContext.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using System.Reactive.Subjects;
+using BTAI;
```
Comment (earlbread) [inline NineChronicles.Standalone/StandaloneContext.cs:2]: This seems to be not used.

File: Libplanet.Standalone.Tests/Hosting/LibplanetNodeServiceTest.cs [hunk 2]
```diff
@@ -5,6 +5,7 @@
 using Libplanet.Action;
 using Libplanet.Blockchain;
 using Libplanet.Blockchain.Policies;
+using Libplanet.Blockchain.Renderers;
```
Comment (earlbread) [inline Libplanet.Standalone.Tests/Hosting/LibplanetNodeServiceTest.cs:8]: This seems to be not used.

File: NineChronicles.Standalone/GraphTypes/StandaloneSubscription.cs [hunk 3]
```diff
@@ -141,22 +143,29 @@ public StandaloneSubscription(StandaloneContext standaloneContext)
 
         public void RegisterTipChangedSubscription()
         {
-            if (StandaloneContext.BlockChain is null)
+            ActionRenderer renderer = StandaloneContext?.NineChroniclesNodeService?.Renderer
+                ?? StandaloneContext.BlockChain.Renderers.OfType<ActionRenderer>().FirstOrDefault();
```
Comment (earlbread) [inline NineChronicles.Standalone/GraphTypes/StandaloneSubscription.cs:147]:
```suggestion
                ?? StandaloneContext?.BlockChain.Renderers.OfType<ActionRenderer>().FirstOrDefault();
```

**Review by riemannulus — APPROVED** [review]

**Comment by dahlia** [comment]
@earlbread Addressed all you reviewed. Please review this again.

**Review by earlbread — APPROVED** [review]

**Pull request merged**